tfe/resource_tfe_team_token.go

```diff
@@ -13,6 +13,9 @@ func resourceTFETeamToken() *schema.Resource {
 		Create: resourceTFETeamTokenCreate,
 		Read:   resourceTFETeamTokenRead,
 		Delete: resourceTFETeamTokenDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
```

Comment:
```
resource "tfe_team_token" "test" {
  team_id          = "${tfe_team.test.id}"
  force_regenerate = false
}
```

ex: `tf import tfe_team_token.test "team-2WbNREnViCFK17hP"`

```
-/+ tfe_team_token.test (new resource required)
      id:                                     "team-2WbNREnViCFK17hP" => <computed> (forces new resource)
      force_regenerate:                       "" => "false" (forces new resource)
      team_id:                                "" => "${tfe_team.test.id}" (forces new resource)
      token:                                  <sensitive> => <computed> (attribute changed)
```

Nothing seems to get set here either.

Reply:
Fixed in the latest version (will push the commit when I'm done).